### PR TITLE
Add Clipboard Support

### PR DESCRIPTION
**Pull request by cyberarm**
Needs to be tested and add swig support

### CODE REVIEW

**Comment by cyberarm** [comment]
⚠️Forgot: Free string with `SDL_free`
 https://wiki.libsdl.org/SDL_GetClipboardText

File: include/Gosu/Input.hpp [hunk 1]
```diff
@@ -107,5 +107,8 @@ namespace Gosu
         TextInput* text_input() const;
         //! Sets the currently active TextInput, or resets it to the nullptr.
         void set_text_input(TextInput* input);
+
+        static void set_clipboard(std::string text);
+        static std::string clipboard();
```
Comment (jlnr) [inline include/Gosu/Input.hpp:112]: Usually the setter comes after the getter...although that's probably not a super consistent convention.

File: src/Input.cpp [hunk 2]
```diff
@@ -678,4 +678,14 @@ void Gosu::Input::set_text_input(TextInput* text_input)
...
 }
 
+void Gosu::Input::set_clipboard(std::string text)
+{
+    SDL_SetClipboardText(text.c_str());
+}
+
+std::string Gosu::Input::clipboard()
+{
+    return std::string(SDL_GetClipboardText());
```
Comment (jlnr) [inline src/Input.cpp:688]:
```suggestion
    std::shared_ptr<char> clipboard{SDL_GetClipboardText(), SDL_free};
    return std::string{clipboard.get()};
```

**Review by jlnr — COMMENTED** [review]
Thanks ✌️

**Comment by cyberarm** [comment]
@jlnr This should be ready, except for the SWIG bits.

**Review by jlnr — APPROVED** [review]

**Comment by jlnr** [comment]
If you want, you can add a unit test for it. (I didn't even get to test it manually...) Otherwise, LGTM, and thanks!

**Comment by cyberarm** [comment]
@jlnr I added a test for `Gosu.clipboard`, if CI passes it should be good to merge! 🥳

**Comment by jlnr** [comment]
Thanks again, merged. :+1:

**Pull request merged**